tests/test_routes.py

```diff
@@ -47,12 +57,29 @@ def test_definition(self):
     response = self.request(url)
     data = json.loads(response.data)
 
-    eq_(data['word'], 'dictionary')
+    eq_(data['metadata']['name'], 'dictionary')
     eq_(data['definitions'], [
-        {"glosses": ["To look up in a dictionary."], "tags": ["transitive"]},
-        {"glosses": ["To add to a dictionary."], "tags": ["transitive"]},
-        {"glosses": ["To compile a dictionary."], "tags": ["rare", "intransitive"]},
-        {"glosses": ["To appear in a dictionary."], "tags": ["intransitive"]}
+        {"metadata": {
+          "description": "To look up in a dictionary.", "pos": "verb", "name": "dictionary",
+          "subject": [{"name": "transitive", "code": "transitive", "schema": "tbd"}],
+          "@type": DictionaryFeed.SCHEMA_DEFINED_TERM, "language": "english"
+        }},
+        {"metadata": {
+          "description": "To add to a dictionary.", "pos": "verb", "name": "dictionary",
+          "subject": [{"name": "transitive", "code": "transitive", "schema": "tbd"}],
+          "@type": DictionaryFeed.SCHEMA_DEFINED_TERM, "language": "english"
+        }},
+        {"metadata": {
+          "description": "To compile a dictionary.", "pos": "verb", "name": "dictionary",
+          "subject": [{"name": "rare", "code": "rare", "schema": "tbd"},
+          {"name": "intransitive", "code": "intransitive", "schema": "tbd"}],
+          "@type": DictionaryFeed.SCHEMA_DEFINED_TERM, "language": "english"
+        }},
+        {"metadata": {
+          "description": "To appear in a dictionary.", "pos": "verb", "name": "dictionary",
+          "subject": [{"name": "intransitive", "code": "intransitive", "schema": "tbd"}],
+          "@type": DictionaryFeed.SCHEMA_DEFINED_TERM, "language": "english"
```

Comment:
The OPDS 2 spec uses 2-letter codes for `language` when it shows up in `metadata`. This may be too big a chance to deal with in this branch, but I'd like to see us use `language` in a standard way. Where does "english" come from here?

Reply:
It's from the url route. I was thinking it should be able to take in a 2-letter code or the full name and then it's internally mapped to a 2-letter code, but yes, it's something to do later on.